todocore/src/test/java/com/knighttodo/todocore/integration/DayTodoResourceIntegrationTest.java

```diff
@@ -99,9 +101,9 @@ public void addDayTodo_shouldAddDayTodoAndReturnIt_whenRequestIsCorrect() throws
         DayTodoRequestDto requestDto = DayTodoFactory.createDayTodoRequestDto();
 
         mockMvc.perform(
-                post(API_BASE_URL_V1 + API_BASE_DAYS + "/" + day.getId() + API_BASE_TODOS)
-                        .content(objectMapper.writeValueAsString(requestDto))
-                        .contentType(MediaType.APPLICATION_JSON_VALUE))
+                        post(API_BASE_URL_V1 + API_BASE_DAYS + "/" + day.getId() + API_BASE_TODOS)
+                                .content(objectMapper.writeValueAsString(requestDto))
+                                .contentType(MediaType.APPLICATION_JSON_VALUE))
```

Comment:
Not related to task. Check if your auto-format style is the same as default  one for the project